cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -0,0 +1,34 @@
+/* topdoc
+  name: Mega Menu
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-mega-menu">
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-mega-menu
+  tags:
+    - cfgov-organisms
+*/
+
+.o-mega-menu {
+
+    // Position and display Submit a Complaint Call to Action (CTA).
+    .m-submit-complaint-cta {
+        display: none;
```

Comment:
Move the display rule to a max-width MQ to avoid resetting to the default on L:25